quantum/quantum.c

```diff
@@ -345,6 +345,25 @@ bool process_record_quantum(keyrecord_t *record) {
             case ONESHOT_DISABLE:
                 oneshot_disable();
                 break;
+#endif
+#if defined(QMK_KEYBOARD) && defined(QMK_KEYMAP)  // tests don't generate these defines
```

Comment:
This should be guarded to allow opt in/out per the other keycodes above. That way there is at least a way to prevent the firmware size increase.

Reply:
Any suggestions on names that aren't absolutely atrocious, such as "no make", "no compile", or variants that start with disable, and/or end with keycode?